cloudmarker/test/test_vmdatadiskencryptionevent.py

```diff
@@ -0,0 +1,153 @@
+"""Tests for VMDataDiskEncryptionEventTest plugin."""
+
+
+import copy
+import random
+import unittest
+
+from cloudmarker.events import vmdatadiskencryptionevent
+
+base_record = {
+    'ext':  {
+        'cloud_type':  'azure',
+        'record_type':  'virtual_machine',
+        'subscription_id':  'sub_id_0000',
+        'all_data_disks_encrypted':  True
+    },
+    'com':  {
+        'cloud_type':  'azure',
+        'record_type':  'compute',
+        'reference':  'reference'
```

Comment:
I don't see any test depending on the `reference` field, so I think this field can also be removed.

Reply:
Removed `reference`